app/src/app/AppNavigator/AppNavigator.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
I don't think prefixing the component name with `App` is necessary. What do you think?